src/renderer/modals/VoteTron/Body.js

```diff
@@ -0,0 +1,207 @@
+// @flow
+import invariant from "invariant";
+import React, { useState, useCallback } from "react";
+import { compose } from "redux";
+import { connect } from "react-redux";
+import { Trans, withTranslation } from "react-i18next";
+import { createStructuredSelector } from "reselect";
+import { SyncSkipUnderPriority } from "@ledgerhq/live-common/lib/bridge/react";
+import Track from "~/renderer/analytics/Track";
+
+import { UserRefusedOnDevice } from "@ledgerhq/errors";
+
+import { getAccountBridge } from "@ledgerhq/live-common/lib/bridge";
+import useBridgeTransaction from "@ledgerhq/live-common/lib/bridge/useBridgeTransaction";
+
+import type { StepId, StepProps, St } from "./types";
+import type { Account, Operation } from "@ledgerhq/live-common/lib/types";
+import type { TFunction } from "react-i18next";
+import type { Device } from "~/renderer/reducers/devices";
+
+import { addPendingOperation } from "@ledgerhq/live-common/lib/account";
+import { updateAccountWithUpdater } from "~/renderer/actions/accounts";
+
+import { getCurrentDevice } from "~/renderer/reducers/devices";
+import { closeModal, openModal } from "~/renderer/actions/modals";
+
+import Stepper from "~/renderer/components/Stepper";
+import StepVote, { StepVoteFooter } from "./steps/StepVote";
+import GenericStepConnectDevice from "~/renderer/modals/Send/steps/GenericStepConnectDevice";
+import StepConfirmation, { StepConfirmationFooter } from "./steps/StepConfirmation";
+import logger from "~/logger/logger";
+
+type OwnProps = {|
+  stepId: StepId,
+  onClose: () => void,
+  onChangeStepId: StepId => void,
+  params: {
+    account: Account,
+    parentAccount: ?Account,
+    reward: number,
+  },
+  name: string,
+|};
+
+type StateProps = {|
+  t: TFunction,
+  device: ?Device,
+  accounts: Account[],
+  device: ?Device,
+  closeModal: string => void,
+  openModal: string => void,
+|};
+
+type Props = {|
```

Comment:
i'm not a flow expert but can you just write like `type Props = OwnProps & StateProps`?

Reply:
probably i copy pasted this part from other modals :sweat_smile: